src/server/api/duplicates.js

```diff
@@ -15,8 +15,8 @@ const dupApi = express();
 const calcSimilarity = (master, other) => {
     let score = 0;
 
-    if (master.first_name == other.first_name) score += 6;
-    if (master.last_name == other.last_name) score += 6;
+    if (master.first_name.trim() == other.first_name.trim()) score += 6;
```

Comment:
This is good. In the future, we might want to optimize this by not repeating it for every iteration. But for the time being we don't really have any performance issues motivating such optimizations.